### ISSUE TITLE
App models need str/unicode attention (Python 3 compatibility)

### ISSUE THREAD

**Issue by caktus-philip** [unicef/etools-issues#746]
All eTools models that implement `__unicode__()` need Django's `@python_2_unicode_compatible` decorator to ensure Python 3 compatibility. 

https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.python_2_unicode_compatible

In addition, each model needs a test that uses non-ASCII characters to exercise `str(instance)` and `unicode(instance)`.

The following apps need attention (checked when PR is merged) --

 - [x] funds (https://github.com/unicef/etools/pull/885)
 - [x] locations (https://github.com/unicef/etools/pull/886)
 - [ ] notification (https://github.com/unicef/etools/pull/892)
 - [ ] partners
 - [ ] publics
 - [ ] reports
 - [ ] supplies
 - [ ] t2f
 - [ ] trips
 - [ ] users
 - [ ] meta test

The last item (meta test) refers to a test in [test_all_models.py](https://github.com/unicef/etools/blob/develop/EquiTrack/EquiTrack/tests/test_all_models.py) that ensures that each model in eTools implements `@python_2_unicode_compatible`. At present, the test is mostly disabled because most app models don't yet implement the decorator, but as apps in the list above are checked off one by one, I'll add them to the meta test. Once the meta test is in place, it should catch any new models that are added to eTools to ensure the project doesn't slip *out* of compatibility.

**Comment by caktus-philip** [comment]
Almost complete! All that remains is for `pro-refactoring` to be merged into `develop` (thus merging #1053 into `develop`) and removing `partners` from the `EXCLUDED_PACKAGES` list in [test_all_models.py](https://github.com/unicef/etools/blob/develop/EquiTrack/EquiTrack/tests/test_all_models.py)
